LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationNetworkServerStartup.cs

```diff
@@ -66,6 +72,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             // TO DO: When certificate generation is properly handled, enable https redirection
             // app.UseHttpsRedirection();
 
+            // Manually set the class C as otherwise the DI fails.
+            var classCMessageSender = app.ApplicationServices.GetService<IClassCDeviceMessageSender>();
+            var dataHandlerImplementation = app.ApplicationServices.GetService<DefaultLoRaDataRequestHandler>();
```

Comment:
We need to get ILoRaDataRequestHandler here, otherwise resolution will fail!

Reply:
With this change, we will also need to put the SetClassCMessageSender in the interface.

Reply:
Done thanks for the test!